tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java

```diff
@@ -316,6 +316,40 @@ public void testInitWithInvalidPackage() throws Exception {
         runMainFunction(projectPath, generatedBalx.toString());
     }
 
+    @Test(description = "Test building a project with a single bal file using the target path")
+    public void testBuildWithTarget() throws Exception {
```

Comment:
Is there two different test scenarios covered from this test method? If yes, we have to move them to two different methods.

Reply:
No only one test scenario is tested if the target file is created at the correct place.